internal/docker/container.go

```diff
@@ -309,6 +316,11 @@ func (r *ContainerRunner) collectResults(results chan result, expected int) bool
 	return passed
 }
 
+func getJobID(jobURL string) string {
```

Comment:
Could you add an example? (https://blog.golang.org/examples)
It's not very clear what a `jobURL` is supposed to look like.

Reply:
sure!